apps/src/code-studio/components/progress/UnitCalendar.story.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import UnitCalendar from './UnitCalendar';
+
+export default storybook =>
```

Comment:
If UnitCalendarDialog is critical to showing the functionality of this component I think you will want the storybook entry to be for that item instead of this one.